src/entities/offerings.ts

```diff
@@ -257,6 +257,7 @@ export interface Product {
   /**
    * Price of the product. This will match the default option's base phase price
    * in subscriptions or the price in non-subscriptions.
+   * @deprecated Use {@link Product.price} instead for cleaner API access.
```

Comment:
I think this could be simpler?
```suggestion
   * @deprecated Use {@link Product.price} instead.
```